package.json

```diff
@@ -30,6 +31,10 @@
     "react-dom": "^16.6.3"
   },
   "dependencies": {
+    "@bigtest/convergence": "^1.1.1",
+    "@bigtest/mirage": "0.0.1",
+    "@folio/stripes-components": "^5.2.1000775",
```

Comment:
Noooo, why are we depending on CI versions of stripes-components and stripes-core here?

Reply:
These should also be dev dependencies, right?

Reply:
I have absolutely no idea how that made it through. My bad. Must be with some copy over when resolving lint issues.